### DISCUSSION TITLE
License Pending Approval

### DISCUSSION THREAD

**Discussion by royaln** [solidify/scale#6] [General]
I am not able to make changes to my process template because the "Start configuration" button is disabled.

**Comment by MOlausson** [comment]
Hi @royaln,

Can you check you have followed the steps in this [guide](https://github.com/solidify/scale/blob/master/docs/Get-started/Installing-Scale.md) and that you have assinged yourself the [extension admin role](https://github.com/solidify/scale/blob/master/docs/Administration/Permissions.md#scale-roles) from the [license management hub](https://github.com/solidify/scale/blob/master/docs/Administration/Manage-licenses.md)?

Mathias

**Reply by royaln** [reply]
Still not working. I just made myself an Extension administrator. As of now, I'm using 2 licenses: 1 as a project administrator and 1 as a extension administrator.

**Reply by MOlausson** [reply]
Can you remove the project admin license so you only have the extension admin? The higher roles are inclusive so you get all the capabilities of the lower role.

**Reply by royaln** [reply]
It's working. Thank you.
